packages/docusaurus-theme-bootstrap/src/theme/ThemedImage/index.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import type {Props} from '@theme/ThemedImage';
+import useThemeContext from '@theme/hooks/useThemeContext';
+
+function ThemedImage(props: Props): JSX.Element {
+  const {isDarkTheme} = useThemeContext();
```

Comment:
useThemeContext does not exist on the bootstrap theme 🤪 

In the meantime, just hardcode isDarkTheme = false with a todo, and I'll probably update this later

Reply:
Personally I think that stubbing the `useThemeContext` in the bootstrap theme would be a better solution - what do you think?

Reply:
Yeah, you can do that too.
The code will be factorized between classic and bootstrap themes at some point, so any solution you find will be thrown away in favor of the classic theme code in the future. Anything that makes the CI happy is good enough for me